plv8_func.cc

```diff
@@ -186,15 +178,6 @@ SubTranBlock::exit(bool success)
 		ReleaseCurrentSubTransaction();
 	else
 		RollbackAndReleaseCurrentSubTransaction();
-
-	MemoryContextSwitchTo(m_mcontext);
-	CurrentResourceOwner = m_resowner;
-
-	/*
-	 * AtEOSubXact_SPI() should not have popped any SPI context, but just
-	 * in case it did, make sure we remain connected.
-	 */
-	SPI_restore_connection();
```

Comment:
thanks - mostly trying to get it all compiling and running with precise, the ubuntu installation travis uses.  i have it compiling, but am getting an undefined symbol at load time.